src/bbs2gh/AwsApiFactory.cs

```diff
@@ -0,0 +1,22 @@
+namespace OctoshiftCLI.BbsToGithub
+{
+    public sealed class AwsApiFactory : IAwsApiFactory
```

Comment:
There is no need to create a `IAwsApiFactory` interfacem the `Create` method can simply be marked as `virtual`